build/dockerfiles/Dockerfile-all

```diff
@@ -0,0 +1,28 @@
+ARG BASE_DOCKER_IMAGE
+FROM ${BASE_DOCKER_IMAGE} as build
+
+RUN mkdir -p "$GOPATH/src/github.com/erda-project/erda/"
+COPY . "$GOPATH/src/github.com/erda-project/erda/"
+WORKDIR "$GOPATH/src/github.com/erda-project/erda/"
+
+RUN make build-all
+
+ARG BASE_DOCKER_IMAGE
+FROM ${BASE_DOCKER_IMAGE}
+
+RUN apk add --no-cache jq
```

Comment:
why install jq?

Reply:
let @recallsong explain it

Reply:
there's a temporary solution:
aim to boost speed of api-test, we make api-test runs in same process as the pipeline.

so jq called by pipeline directly...

and we are proposing the final solution, will be fixed in the next iteration.

Reply:
@recallsong

Reply:
> there's a temporary solution:
> aim to boost speed of api-test, we make api-test runs in same process as the pipeline.
> 
> we use jq to achieve some function like value-extracting, assertion... so jq called by pipeline directly...
> 
> and we are proposing the final solution, will be fixed in the next iteration.

got it